src/components/menus/directory-tree-contextual-menu.tsx

```diff
@@ -411,7 +411,12 @@ export default function DirectoryTreeContextualMenu(props: Readonly<DirectoryTre
     return (
         <>
             {open && (
-                <CommonContextualMenu {...others} menuItems={buildMenu()} open={open && !hideMenu} onClose={onClose} />
+                <CommonContextualMenu
+                    {...others}
+                    menuItems={buildMenu()}
+                    open={open && !hideMenu}
+                    onClose={() => onClose()}
```

Comment:
Why not just pass onClose, it's already a callback

Reply:
TS2322: Type
(nextSelectedDirectoryId?: string | null | undefined) => void
is not assignable to type
(event: {}, reason: "backdropClick" | "escapeKeyDown") => void
Types of parameters nextSelectedDirectoryId and event are incompatible.
Type {} is not assignable to type string
Menu. d. ts(52, 3): The expected type comes from property onClose which is declared here on type
IntrinsicAttributes & Readonly<CommonContextualMenuProps>

This is a proposition of Quentin as a hack on typing